src/content/zh-tw/fundamentals/accessibility/index.md

```diff
@@ -13,127 +12,82 @@ description:改善網頁的無障礙功能
 {% include "web/_shared/contributors/aliceboxhall.html" %}
 {% include "web/_shared/contributors/robdodson.html" %}
 
-
 本文是 [Udacity 無障礙功能課程](https://www.udacity.com/course/web-accessibility--ud891){: .external }中所涉及部分內容的文字版本。但其並非視頻課程的直接文字記錄，而是旨在以課程原有內容爲基礎，對無障礙功能的原理和實踐進行更簡要的論述。
 
-
-
-
-
-
 ### TL;DR {: .hide-from-toc }
+
 - 瞭解無障礙功能的含義及其對網絡開發的作用。
 - 瞭解如何讓所有人都能訪問和使用網站。
 - 瞭解如何在儘量減小開發影響的情況下加入基本無障礙功能。
 - 瞭解提供了哪些 HTML 功能以及如何利用它們來改善無障礙功能。
 - 瞭解在打造美觀的無障礙體驗時一些高級無障礙功能技巧的相關內容。
 
-
-
 瞭解無障礙功能及其範圍和影響可令您成爲更出色的網絡開發者。
 本指南旨在幫助您瞭解如何才能讓所有人都能訪問和使用您的網站。
 
-
 儘管“Accessibility”（無障礙功能）難以拼寫，卻不一定難以實現。
 在本指南中，您將瞭解如何通過一些速效方法以最少的工作量幫助改善無障礙功能，如何才能利用 HTML 的內置功能打造更加強健並具有更佳無障礙功能的界面，以及如何充分利用一些高級技巧來打造美觀的無障礙體驗。
 
-
-
-
 您還會發現，其中的許多技巧幫助您打造的界面所帶來的滿意度和易用性提升並非僅僅適用於殘障人士，而是適用於*所有*用戶。
 
-
-
 當然，許多開發者對無障礙功能含義的理解還很模糊（不就是與政府合同、檢查清單和屏幕閱讀器有關的功能嗎？），也流傳着許多有關它的誤解。例如，許多開發者認爲，處理無障礙功能問題就是強迫他們在打造賞心悅目的體驗與粗鄙醜陋但具有無障礙功能的體驗之間做出選擇。
 
-
-
-
-
-
 當然，實際情況根本不是這樣，因此讓我們先澄清這個問題，然後再進入正題。
 無障礙功能有何含義？您可以通過本文了解它的哪些相關內容？
 
-
 ## 什麼是無障礙功能？
-一般來說，當我們說某個網站具有無障礙功能時，我們的意思是網站的內容可用，並且毫不誇張地講，其功能可由*任何人*操作。作爲開發者，您會輕易地認爲所有用戶都能看見和使用鍵盤、鼠標或觸摸屏，並且與網頁內容的交互方式也與您相同。這會造成一些人能夠獲得良好的體驗，其他人則會遇到從簡單煩惱到重大障礙的各種問題。
-
 
+一般來說，當我們說某個網站具有無障礙功能時，我們的意思是網站的內容可用，並且毫不誇張地講，其功能可由*任何人*操作。作爲開發者，您會輕易地認爲所有用戶都能看見和使用鍵盤、鼠標或觸摸屏，並且與網頁內容的交互方式也與您相同。這會造成一些人能夠獲得良好的體驗，其他人則會遇到從簡單煩惱到重大障礙的各種問題。
 
 那麼，無障礙功能就是指這樣一類用戶的體驗，他們可能不在“典型”用戶這一狹窄範圍之內，並且與網站的訪問或交互方式異於常規。具體地講，它所涉及的用戶具有某種身體缺陷或殘障（切記這種體驗可能具有非生理性或暫時性）。
 
-
-
 例如，儘管我們在探討無障礙功能時往往是圍繞身體有缺陷的用戶，但實際上都能將其與我們使用的界面由於其他原因而無法訪問的經歷聯繫起來。您是否曾在手機上使用桌面版本網站時遇到問題？是否看到過“您所在地區不提供該內容”消息？或者是否在平板電腦上找不到熟悉的菜單？這些都屬於無障礙功能問題。
 
 隨着瞭解的深入，您會發現在這種更廣泛、更普遍意義上解決無障礙功能問題幾乎總能讓所有人的用戶體驗得到改善。
 
-下面我們來看一個示例：
+讓我們來看一個例子：
 
-![一個無障礙功能不佳的表單](imgs/pooraccess.jpg)
+![a form with poor accessibility](imgs/pooraccess.jpg)
```

Comment:
@kennith Is it possible to translate alt tag for it? GitLocalize does not have the feature. Thanks in advance.